src/test/java/edu/cornell/kfs/coa/fixture/A21IndirectCostRecoveryAccountFixture.java

```diff
@@ -0,0 +1,40 @@
+package edu.cornell.kfs.coa.fixture;
+
+import java.math.BigDecimal;
+
+import org.kuali.kfs.coa.businessobject.A21IndirectCostRecoveryAccount;
+
+public enum A21IndirectCostRecoveryAccountFixture {
+	A21_ICR_1111111_98_PERCENT_ACTIVE("IT", "1111111",  new BigDecimal(98), true),
+	A21_ICR_1111111_98_PERCENT_INACTIVE("IT", "1111111",  new BigDecimal(98), false),
+	A21_ICR_2222222_2_PERCENT_ACTIVE("IT", "2222222",  new BigDecimal(2), true),
+	A21_ICR_2222222_2_PERCENT_INACTIVE("IT", "2222222",  new BigDecimal(2), false),
+	A21_ICR_3333333_100_PERCENT_ACTIVE("IT", "3333333",  new BigDecimal(100), true),
+	A21_ICR_3333333_100_PERCENT_INACTIVE("IT", "3333333",  new BigDecimal(100), false);
+	
+    public String indirectCostRecoveryFinCoaCode;
+    public String indirectCostRecoveryAccountNumber;
+    public BigDecimal accountLinePercent;
+    public boolean active;
```

Comment:
In this fixture and all the other new fixtures being added, I would strongly recommend declaring these fields as "final", to prevent testing code from accidentally updating them.